### PR TITLE
feat: staking banner

### PR DESCRIPTION
**Pull request by ChefJerry**
---

## PR-Codex overview
The focus of this PR is to update the banners in the web app, adding a new WBNB fixed staking banner with graphics and actions.

### Detailed summary
- Added a new `WBNBFixedStakingBanner` component
- Updated translations in `translations.json`
- Updated styles and content in `BannerGraphics.tsx` and `BannerMain.tsx`
- Added new imports and components for the banner in various files

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx [hunk 1]
```diff
@@ -41,6 +42,10 @@ export const useMultipleBannerConfig = () => {
   const countdown = useCountdown(1704369600)
   return useMemo(() => {
     const NO_SHUFFLE_BANNERS: IBannerConfig[] = [
+      {
+        shouldRender: true,
+        banner: <WBNBFixedStakingBanner />,
+      },
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:48]: Move it under `UserBanner` & `NewIFOBanner`

**Comment by ChefMomota** [comment]
Screenshot 2024-03-26 at 5 51 36 PM

**Review by ChefMomota — DISMISSED** [review]

File: apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      <BannerMain
+        badges={<PancakeSwapBadge />}
+        title={
+          <BannerTitle variant="purple">
+            {isMobile ? t('Stake WBNB, Win Combo Rewards!') : t('Stake WBNB to Win Combo Rewards')}
+          </BannerTitle>
+        }
+        desc={isMobile ? null : <Desc />}
+        actions={
+          <Actions>
```
Comment (chefjackson) [inline apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx:146]: Let's use https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/widgets-internal/components/Banner/BannerActionContainer.tsx#L4

Comment (ChefJerry) [reply apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx:146]: the `BannerActionContainer` can not use at this case, the actions should be `postion: absolute`

File: apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            {stakeAction}
+            {isMobile ? null : <VerticalDivider />}
```
[the remark this answers is not in the document]
Comment (ChefJerry) [reply apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx:148]: not sure if it need to be a reused component, as it just a styled border line

File: apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx (hunk 2 above)
Comment (chefjackson) [reply apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx:146]: why do we need position absolute?

File: apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (chefjackson) [reply apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx:148]: We may need that on title as well

File: apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx (hunk 2 above)
Comment (ChefJerry) [reply apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx:146]: image

the actions will have vertical stack on bg

**Review by chefjackson — APPROVED** [review]

**Review by chefjackson — APPROVED** [review]

**Pull request merged**